### PR TITLE
Switch to SPDX license name

### LINKED ISSUE #61: Use SPDX identifier for license
- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue: The license is specified in freeform. Should be replaced by the corresponding SPDX identifier.
Unfortunately, the license situation is not quite straight-forward: The link in the meta.yaml points to a 3 clause BSD, but I belief this is an error due to a license change of the upstream netcdf library.
The correct license for netcdf-fortran seems to still be the old netcdf license (SPDX: [NetCDF](https://spdx.org/licenses/NetCDF.html)) according to the packaged COPYRIGHT file.
Furthermore, that COPYRIGHT file also contains a statement about some fortran code being under Apache-2, but it is not apparent which parts that would apply to.

Not sure what to do.

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

closes #61

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
~~Not bumping build number because this is just to fix the license name.~~

Update: need to bump after all.

**Comment by xylar** [comment]
Thanks @zklaus for figuring this out.

**Comment by xylar** [comment]
That's not promising! MPICH builds are failing for both Linux configurations.

**Comment by xylar** [comment]
Works fine in a local build, which makes debugging much harder...

**Comment by xylar** [comment]
Anyone know how to get more verbose output from the tests here?

**Comment by zklaus** [comment]
It seems like it's only the tests that are failing. I would probably try commenting out the tests in `build.sh` to confirm. If that is indeed the case, try adding [`-VV`](https://cmake.org/cmake/help/latest/manual/ctest.1.html#options) to the `ctest` call in `build.sh`.

**Comment by xylar** [comment]
It's definitely just the tests.  I'll add `-VV`.

**Comment by xylar** [comment]
@zklaus, thanks for the advice.  Let's see what happens.  Presumably, you'll need this for #60

**Comment by xylar** [comment]
The issue seems to be with not find `libquadmath`:
```
1: Test command: $SRC_DIR/build_shared/nf_test/nf_test
1: Test timeout computed to be: 1500
1: $SRC_DIR/build_shared/nf_test/nf_test: error while loading shared libraries: libquadmath.so.0: cannot open shared object file: No such file or directory
```

But that's very odd because this should come along with `mpich` through the standard c libraries.  And it used to work!  And it works fine for the other (no)MPI flavors!

File: recipe/build.sh [hunk 1]
```diff
@@ -55,5 +55,9 @@ cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
       $SRC_DIR
 
 make
-ctest
+if [[ $(uname) == Linux ]]; then
+  # seems to be needed to find libquadmath.so.0
+  export LD_LIBRARY_PATH="$PREFIX/lib:$LD_LIBRARY_PATH"
+fi
```
Comment (xylar) [inline recipe/build.sh:61]: Anyone have any problem with this fix?  Odd that it's needed but seems to work.

Comment (xylar) [reply recipe/build.sh:61]: @conda-forge/netcdf-fortran, any comments?

**Comment by xylar** [comment]
I'm going to go ahead with this.  Hopefully, users don't see this problem.  But even if so, I don't think this PR will introduce any problems that aren't already there.

**Pull request merged**